src/object.d

```diff
@@ -4506,4 +4506,5 @@ template _arrayOp(Args...)
     alias _arrayOp = arrayOp!Args;
 }
 
-void __ctfeWrite(scope const(char)[] s) @nogc @safe pure nothrow {}
+/// Legacy import
+public import core.builtins : __ctfeWrite;
```

Comment:
Should this be deprecated if legacy? Or I guess there is no need to punish users in this case.

Reply:
I think we should be fine without a deprecation in this case.

Reply:
This should stay IMHO, It's nice to have `__ctfeWrite` available without having to search for the right module to import.

Reply:
The import is a good idea, but the comment "legacy import" is not, as it implies that the import exist for backwards compatibility reasons (which is not true, since this wasn't implemented until now). Removing it (while keeping the import) will reduce confusion.

Reply:
Fair enough. I wasn't sure whether this import would be deprecated or not when I wrote this comment.

=> See #3437